superset/models/helpers.py

```diff
@@ -321,11 +322,11 @@ def template_params_dict(self) -> Dict[Any, Any]:
         return json_to_dict(self.template_params)  # type: ignore
 
 
-def _user_link(user: User) -> Union[Markup, str]:  # pylint: disable=no-self-use
-    if not user:
+def _user_link(_user: User) -> Union[Markup, str]:
```

Comment:
Can't we just remove the `# pylint: disable=no-self-use` on this one?

Reply:
You are absolutely right! Somehow I had to mistake that for `unused-argument`. Thanks! I'm changing it right-away.